### ISSUE TITLE
Gold-bar when opening a connected solution

### ISSUE THREAD

**Issue by valhristov** [SonarSource/sonarlint-visualstudio#809]
### Description

A gold bar appears when a solution is opened and SonarLint is installed. See the following thread for some details and screenshots:
https://community.sonarsource.com/t/sonarlint-extension-unresponsive-vs-2017/6468

Possible reason could be a large number of suppressed issues on SonarQube. In this case the calls to the web service should be performed on the background thread so that we avoid freezing the UI.

**Comment by rita-g-sonarsource** [comment]
Tested on all VS versions (15, 17, 19) with connection to SonarCloud and over 24k supressions.
Could not reproduce.
Closing due to lack of details.

**Comment by IT-CASADO** [comment]
Same for me on opening my solution in Visual Studio 2019.
**Everytime** I open my solution (around 80 C# projects and 1 database project) I need to wait for 70 - 90 Seconds.

![image](https://user-images.githubusercontent.com/2911693/119616393-d6ebff00-be00-11eb-8915-edd9d15d0383.png)

![image](https://user-images.githubusercontent.com/2911693/119616477-f2570a00-be00-11eb-9021-d51c0bc01643.png)


Output Info:

```
Initializing the telemetry package...
Current VS version:
    Visual Studio Enterprise 2019
    16.9.31313.79
    16.9.6
Finished initializing the telemetry package...
Initializing the daemon package...
Settings file does not exist at "C:\Users\CasadoDaniel\AppData\Roaming\SonarLint for Visual Studio\settings.json".
Default settings will be used
[Open in IDE] Creating request listener...
[Open in IDE] Checking availability of port 64120
[Open in IDE] Port 64120 is unavailable
[Open in IDE] Checking availability of port 64121
[Open in IDE] Request listener created successfully. Listening on port 64121.
Finished initializing the daemon package.
Connecting to 'https://sonarcloud.io/'.
Connected to SonarCloud '8.0.0.20278'.
Suppressions have been updated. Open documents will be re-analysed.
Starting job: "re-analyzing 0 document(s)...". Time: 06:57:51
Finished job "re-analyzing 0 document(s)..." started at 06:57:51. Elapsed time: 0ms

******  BETWEEN THOSE LINES VERY LONG PAUSE (THIS SEEMS THE BOTTLENECK ********

Checking for unbound projects.

******  FOR EVERY PROJECTS (VERY FAST) ********

Project MyProject RuleSet declarations:
   Configuration: Debug, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'
   Configuration: Debug, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'
   Configuration: Release, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'
   Configuration: Release, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'
   Configuration: API, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'
   Configuration: API, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'
   Configuration: UI, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'
   Configuration: UI, RuleSetPath: '..\..\.sonarlint\evocsharp.ruleset', RuleSetDirectories: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Team Tools\Static Analysis Tools\\Rule Sets'

No unbound projects were found.
Checking if SonarQube Quality profile has changed.
Checking for suppressions...
The Quality Profile for the SonarQube project has changed. Update is required.
Number of suppressions found: 61
Initializing the SonarLint package...
Connected mode detected.
Finishing initializing the SonarLint package
Initializing the notifications package...
Loading notifications settings...
Connected: checking for notifications
Finished initializing the notifications package
[Taint] Initializing taint issues synchronization package...
[Taint] Fetched 0 taint vulnerabilities.
[Taint] Finished initializing taint issues synchronization package.


```

**Comment by rita-sonarsource** [comment]
Thank you for additional info @IT-CASADO, reopening.

**Comment by duncanp-sonar** [comment]
Closing a likely duplicate of #2791, fixed in v5.2.
